pil-analyzer/src/condenser.rs

```diff
@@ -36,14 +36,13 @@ use crate::{
     statement_processor::Counters,
 };
 
-type ParsedIdentity = Identity<parsed::SelectedExpressions<Expression>>;
 type AnalyzedIdentity<T> = Identity<SelectedExpressions<AlgebraicExpression<T>>>;
```

Comment:
Given that there is only one type of identity now, does it make sense to keep this? Or is it because clippy complains about the length?

Reply:
I think it makes sense, yes. I wanted to rename it to Identity but of course then we get a collision. What we could do is change the definition if Identity to be non-generic, but I would say that's another PR.